src/components/param-ui/fieldset-loader.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState, useContext } from 'react';
+
+import useSchema from '../../hooks/use-schema';
+import useSchemaLoader from '../../hooks/use-schema-loader';
+import { checkIncludesPath, hasSetEntry, toggleSetEntry } from '../../utils';
+import { LocationContext } from '../../contexts/location';
+import { textDisabled } from '../../lib/messages';
+import ParamSelect from './param-select';
+import { Add, Done } from '../icon';
+
+const Attribute = ({ attribute, type, includeEnabled }) => {
+  const { fields, toggleField } = useContext(LocationContext);
```

Comment:
Since we're already using `LocationContext` in this component, I don't think it still makes sense to pass `includeEnabled` as a prop.